daisy/step_create_instances.go

```diff
@@ -134,7 +140,7 @@ func (c *CreateInstances) run(ctx context.Context, s *Step) DError {
 				eChan <- newErr("failed to create instances", err)
 				return
 			}
-			go logSerialOutput(ctx, s, i, 1, 3*time.Second)
+			go logSerialOutput(ctx, s, i, 1, 5*time.Second)
```

Comment:
Why the change to 5 second refresh interval?

Reply:
Three felt a bit spammy. On the other hand, it's been working for awhile, so, it's probably higher risk to change it at this point (since when an instance stops we lose access to its serial logs, so that would mean potentially losing two extra seconds of logs)

Fixed back to three.